packages/validator/test/cli-validator/mock-files/oas3/err-and-warn.yaml

```diff
@@ -173,6 +173,7 @@ components:
           description: string
         category:
           $ref: "#/components/schemas/Category"
+          description: 'This ref sibling should cause an error'
```

Comment:
Just added one violation of the no-$ref-siblings rule in both flavors (3.0, 3.1) of this testcase.